test/wallaby/browser/local_storage_test.exs

```diff
@@ -16,24 +16,32 @@ defmodule Wallaby.Browser.LocalStorageTest do
     |> visit("index.html")
     |> execute_script(@set_value_script)
 
-    assert session
-    |> execute_script(@get_value_script) == "foo"
+    session
+    |> execute_script(@get_value_script, fn(value) -> send self(), {:result, value} end)
+
+    assert_received {:result, "foo"}
```

Comment:
any reason against using the assert straight inside the callback function to avid the send work around?